django/contrib/sitemaps/management/commands/ping_google.py

```diff
@@ -7,6 +7,12 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('sitemap_url', nargs='?')
+        parser.add_argument('-d', '--site_domain', default=None)
+        parser.add_argument('--use_http', default=False, action='store_false')
 
     def handle(self, *args, **options):
-        ping_google(sitemap_url=options['sitemap_url'])
+        ping_google(
+            sitemap_url=options['sitemap_url'],
+            site_domain=options['site_domain'],
+            is_http=options['use_http']
```

Comment:
Include a trailing comma in cases like this so that if more items are added later, this line doesn't have to be modified again.